src/containers/header/Header.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import styled from 'styled-components';
+import img from '../../static/header-background.jpg';
+import HeaderContentSearch from '../header-content-search/HeaderContentSearch';
+import HeaderContentMovie from '../header-content-movie/HeaderContentMovie';
+import { withRouter } from 'react-router-dom';
```

Comment:
it's better to place absolute paths before relative

Reply:
And the same mention about to use hooks. https://reacttraining.com/react-router/web/api/Hooks/useroutematch